lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -218,5 +218,6 @@ def video_summary(course, course_id, video_descriptor, request, local_cache):
         "transcripts": transcripts,
         "language": video_descriptor.get_default_transcript_language(),
         "category": video_descriptor.category,
+        "only_on_web" : video_descriptor.unavailable_on_mobile,
```

Comment:
Actually, my first comment is wrong. The wording is pretty confusing and I now just realized that `unavailable_on_mobile` = `True` and `only_on_web` = `True` mean the same thing. Can we call `only_on_web` also `unavailable_on_mobile`?